samples/bluetooth/bap_broadcast_assistant/prj.conf

```
@@ -1,6 +1,7 @@
 CONFIG_BT=y
 CONFIG_LOG=y
 CONFIG_BT_CENTRAL=y
+CONFIG_BT_OBSERVER=y
```

The observer is implied for centrals
```suggestion
CONFIG_BT_CENTRAL=y
```

---

That's true. I'll correct it.